src/main/webapp/jsp/adminAccountManagement.jsp

```diff
@@ -1,216 +1,46 @@
-                                class="btn btn-link btn-xs "> <span
-                                    class="glyphicon glyphicon-trash"></span>
-                                    Delete Entire Account
-                            </a>
-
-                                <form method="post" target="blank"
-                                    action="<%=Const.ActionURIs.ADMIN_ACTIVITY_LOG_PAGE%>">
-                                    <button type="submit" 
-                                        id="<%=acc.googleId + "_recentActions"%>"
-                                        class="btn btn-link btn-xs">
-                                        <span
-                                            class="glyphicon glyphicon-zoom-in"></span>
-                                        View Recent Actions
-                                    </button>
-                                    <input type="hidden"
-                                        name="filterQuery"
-                                        value="person:<%=acc.googleId%>">
-                                </form>
-                        </tr>
-                        <%
-                        	}
-                        %>
-                    </tbody>
-
-                </table>
-            </div>
-        </div>
-
-        <div id="pagination_bottom">
-            <ul class="pagination">
-                <li class="previous"><a href="#"> <span>&laquo;</span>
-                </a></li>
-                <li><a class="pageNumber" href="#">1</a></li>
-                <li><a class="pageNumber" href="#">2</a></li>
-                <li><a class="pageNumber" href="#">3</a></li>
-                <li><a class="pageNumber" href="#">4</a></li>
-                <li><a class="pageNumber" href="#">5</a></li>
-                <li class="next"><a href="#"><span>&raquo;</span>
-                </a></li>
-            </ul>
-        </div>
-
-
-        <a href="#" class="back-to-top-left"><span
-            class="glyphicon glyphicon-arrow-up"></span>&nbsp;Top</a> 
-            
-            <a
-            href="#" class="back-to-top-right">Top&nbsp;<span
-            class="glyphicon glyphicon-arrow-up"></span></a>
-
+</c:set>
+
+<ta:adminPage bodyTitle="Instructor Account Management" pageTitle="TEAMMATES - Administrator Account Management" jsIncludes="${jsIncludes}">    <t:statusMessage/>
+    <div id="pagination_top">
+        <ul class="pagination">
+            <li class="previous"><a href="#"> <span>&laquo;</span>
+            </a></li>
+            <li><a class="pageNumber" href="#">1</a></li>
+            <li><a class="pageNumber" href="#">2</a></li>
+            <li><a class="pageNumber" href="#">3</a></li>
+            <li><a class="pageNumber" href="#">4</a></li>
+            <li><a class="pageNumber" href="#">5</a></li>
+            <li class="next"><a href="#"><span>&raquo;</span>
+            </a></li>
+        </ul>
```

Comment:
Since this block is repeated below, you can put it in a tag called adminAccountManagementPagination.tag